src/MunicipalityRegistry.Api.Legacy/Municipality/MunicipalityController.cs

```diff
@@ -316,13 +324,23 @@ private static IEnumerable<GeografischeNaam> GetGemeentenamenByLanguage(Municipa
             return sw.ToString();
         }
 
-        private static Uri BuildVolgendeUri(PaginationInfo paginationInfo, string volgendeUrlBase)
+        private static Uri BuildNextUri(PaginationInfo paginationInfo, string nextUrlBase)
         {
             var offset = paginationInfo.Offset;
             var limit = paginationInfo.Limit;
 
             return paginationInfo.HasNextPage
-                ? new Uri(string.Format(volgendeUrlBase, offset + limit, limit))
+                ? new Uri(string.Format(nextUrlBase, offset + limit, limit))
+                : null;
+        }
+
+        private static Uri BuildNextSyncUri(PaginationInfo paginationInfo, long? from, string nextUrlBase)
+        {
+            var offset = paginationInfo.Offset;
+            var limit = paginationInfo.Limit;
+
+            return from.HasValue
+                ? new Uri(string.Format(nextUrlBase, from.Value + 1, limit))
```

Comment:
why **from + 1**, if i pass **from=100** I would expect to get position 100 too

Reply:
I think this is to build the next url. You ask from=100 and get 100-200, and the next url will be from=201?

Reply:
But if that is the case, it should probable be from + limit + 1

Reply:
`from` is inclusive. So yes, `from = 100` will give you event 100.
However the naming does not reflect the intent.